projects/dsp-ui/src/lib/action/components/login-form/login-form.component.ts

```diff
@@ -159,6 +161,10 @@ export class LoginFormComponent implements OnInit {
                 this.loginFailed = (error.status === 401 || error.status === 404);
                 this.loginErrorServer = (error.status === 0 || error.status >= 500 && error.status < 600);
 
+                if (this.loginErrorServer) {
+                    this._notification.openSnackBar(error);
+                }
+
                 this.loginSuccess.emit(false);
                 this.errorMessage = error;
```

Comment:
I think `errorMessage` can be simplified to a boolean now and renamed. The template only checks if it's defined.

Reply:
Done in 7bec4cd.